sectionTwoCode/class20.txt

```diff
@@ -0,0 +1,10 @@
+//1, 2, 3
+//2, 4, 6
+
+const numbers = [1, 2, 3];
+const doubleNumArray = numbers.map((num) => {
```

Comment:
Get to know what is => ?

Reply:
An arrow function expression is a compact alternative to a traditional function expression, but is limited and can't be used in all situations.

Differences & Limitations:

Does not have its own bindings to this or super, and should not be used as methods.
Does not have new.target keyword.
Not suitable for call, apply and bind methods, which generally rely on establishing a scope.
Can not be used as constructors.
Can not use yield, within its body.